src/dsp_tools/commands/ingest_xmlupload/create_resources/upload_xml.py

```diff
@@ -67,15 +69,21 @@ def ingest_xmlupload(
     clients = _get_live_clients(con, config)
     state = UploadState(resources, stash, config, permissions_lookup)
 
-    return execute_upload(clients, state)
+    if not execute_upload(clients, state):
+        success = False
 
+    if not _tidy_up_on_server(con, shortcode, creds.dsp_ingest_url):
+        success = False
 
-def _parse_xml(xml_file: Path) -> tuple[str, etree._Element, str]:
+    return success
+
+
+def _parse_xml_and_replace_filepaths(xml_file: Path) -> tuple[str, etree._Element, str]:
     """
-    Validate and parse an upload XML file, when preprocessing has already been done.
+    Validate and parse an upload XML file, when ingest has already been done.
```

Comment:
Maybe say that ingest is a prerequisite. Otherwise I could be lead to believe that it is an automated step that occurs before.

Reply:
Done with https://github.com/dasch-swiss/dsp-tools/commit/fb080db2e118ba1e1b66cd18cbcc8ed55222bcae